### ISSUE TITLE
RMSNorm with bias kwarg option

### ISSUE THREAD

**Issue by PeaBrane** [pytorch/pytorch#138025]
### 🚀 The feature, motivation and pitch

Currently, the [RMSNorm module](https://pytorch.org/docs/stable/generated/torch.nn.modules.normalization.RMSNorm.html) supports the `elementwise_affine` kwarg but not the `bias` kwarg. Would it be easy to implement a mode where `elementwise_affine` is set True, but `bias` can be set False, like [LayerNorm](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html)?

This sort of configuration is fairly prevalent and is used by many modern LLM models including Llama(s), and Mamba(s). 

cc @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

**Comment by PeaBrane** [comment]
I actually realized just now that the current behavior of nn.RMSNorm is to not apply a bias, which actually fits the common usage. This is probably good to close, unless there is a popular usage where bias=True for RMSNorm (which I'm currently not aware)
